.vscode/settings.json

```diff
@@ -1,4 +1,10 @@
 {
+  "eslint.options": {
+    "rulePaths": [
+      "./eslint"
+    ]
+  },
```

Comment:
I think this shouldn't be needed.  I changed this to be done through a plugin instead so that it would work without options.